app/models/shopify_data_feed.rb

```diff
@@ -0,0 +1,5 @@
+class ShopifyDataFeed < ActiveRecord::Base
```

Comment:
I'm not expert either. I think the idea of namespaces is to avoid collision with other gems/engines. And the rule should be that each gem has unique namespace, mostly denoted by gem's name. Having Shopify namespace in one place, no namespace in the other, Spree namespace in third place,... Looks a bit messy to me. What if guys from Shopify introduce next version of their API and it's namespaced like this:

```rb
module Shopify
  # client for connecting to Shopify API
  class Client
  end
end
```